hms-app/src/actions/ParticipantActions.tsx

```diff
@@ -37,3 +37,20 @@ export const createParticipant = (userId: string, hackathonId: string) => {
     })
     .catch((err) => console.log(err))
 }
+
+export const getParticipantDetails = (participantID: string) => {
+  return fetch(
+    `http://localhost:4566/restapis/${api_id}/local/_user_request_/category/${participantID}`,
```

Comment:
so this `http://localhost:4566/restapis/${api_id}/local/_user_request_` is repeated a lot.  You could probably extract it as a a const named something like hmsApiBaseUrl and put it where the thing that you get the `api_id` is.  Just cuts down on duplication and shortens these lines